share/translations/keepassxc_en.ts

```diff
@@ -6888,6 +6888,14 @@ Do you want to overwrite it?</source>
         <source>Special Characters</source>
         <translation type="unfinished"></translation>
     </message>
+    <message>
+        <source>passwordLength</source>
```

Comment:
This is a translatable string?

Reply:
Oops yah that should be marked not translated